### ISSUE TITLE
Albums with the same name are aggregated

### ISSUE THREAD

**Issue by MineyMe** [gnumdk/lollypop#44]
Hello

I'm completely new to Lollypop So far I'm liking it; I was looking forward to gnome-music as well but its performance is abysmal (and it is every startup) with my fairly large music library; lollypop doesn't suffer any problems and is 100% responsive and snappy here.

It seems that Lollypop is aggregating all the albums I have that happen to be titled 'Greatest Hits' under one large album. The 'Greatest Hits' album is also not listed under the corresponding artist. Using the search functionality to play one of the missing tracks and the 'View albums of the same artist' view in the header bar (to the left of the seek slider), I see that it was put under the same album as the 'Greatest Hits' album from another artist (along with all other albums of the same name).

I don't know how the code works, but perhaps it would be a solution to include the 'artist name' as criteria for searching tracks under the same album?

Thanks in advance!

**Comment by gnumdk** [comment]
From my quick testing, i think your albums are not correctly tagged...

Two albums with same name but different artist name are compilations (same behaviour as Amarok and Clementine).

If you want to have different albums, install easytag and set the "album artist" tag

**Comment by MineyMe** [comment]
I see, if I understand correctly this behavior is actually to take into account compilation CD's (e.g. 90's greatest hits) containing several artists and have them become bundled into one actual CD, even though the artist names are different. Then if a certain artist would release a Greatest Hits CD, he would not only be the 'Artist' on every one of the tracks, he would also be the 'Album artist'.

In essence, in a bunch of tracks with random artists and albums, the tracks are first grouped by album name and then subsequently grouped by album artist, the latter which is missing in my tags. I've tried setting the album artist tag of one of the "wrongly" placed albums and this seems to work, I had to retag quite a few albums to make the genres consistent anyway, thanks!
